build_keyboard.mk

```diff
@@ -223,6 +225,27 @@ ifneq ($(strip $(MAX_VARIABLE_TRACE_SIZE)),)
 endif
 endif
 
+ifeq ($(strip $(LCD_ENABLE)), yes)
+CIE1931_CURVE = yes
```

Comment:
Nit: might as well be consistent about indentation here and below... some of these blocks are indented, some aren't.